user/useradm.go

```diff
@@ -470,7 +476,7 @@ func (ua *UserAdm) Verify(ctx context.Context, token *jwt.Token) error {
 	}
 
 	//check service-specific claims - iss
-	if token.Claims.Issuer != ua.config.Issuer {
+	if _, ok := ua.config.AllowedIssuers[token.Claims.Issuer]; !ok {
```

Comment:
This will break current installations if you don't set `allowed_jwt_issuers`, won't it?

Reply:
this is something Alf requested. I chose to set the default issuer here:https://github.com/mendersoftware/useradm/pull/334/files#r1016168473